imports/ui/signup/signup.jsx

```diff
@@ -129,6 +100,13 @@ export class SignupPage extends Component {
             ))}
             <PasswordInput value={info.password} onChange={evt => this.makeInfoChange({password: evt.target.value})} />
           </fieldset>
+          <div className='f7 gray mt3 lh-copy'>
+            <label className='pa0 ma0 lh-copy f6 pointer mid-gray'>
+              <input type='checkbox' onChange={this.handleTermCheckbox} />
```

Comment:
The checkbox is not bound to the value on the state object
(There's a missing attribute: 
```
checked={...}
```
)